eve/service/svc_note.py

```diff
@@ -0,0 +1,172 @@
+import json
+from rich.console import Console
+from rich.table import Table
+from rich import box
+import os.path
+from datetime import date
+import pathlib
+
+
+class Note:
+    def __init__(self):
+        # path to JSON-file where to store note-data
+        self.path = str(pathlib.Path.home()) + '/test.json'  # rename for production and make it a hidden file
+
+        # check if JSON-file already exists and if not create it with sample entries
+        if os.path.isfile(self.path):
+            pass
```

Comment:
Not sure if this will be in the final version but there is almost never a reason to use the `pass` keyword in python unless you're testing something. In this case you're better to do;
```python
if not os.path.isfile(self.path):
    # do my thing
```

Reply:
I always do this. In my mind every if-statement has to have an else-statement too. It's bad habit. Thanks for mentioning.